src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.VisualBasic.Analyzers.targets

```
@@ -0,0 +1,29 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.VisualBasic.Analyzers.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(EnableNETAnalyzers )' == '' And 
+                            '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And
+                             $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '5.0'))">
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+  </PropertyGroup>
+  <Import Project="$([System.IO.Path]::GetFullPath($(MSBuildThisFileDirectory)..))\analyzers\build\Microsoft.CodeAnalysis.NetAnalyzers.targets"
+          Condition="$(EnableNETAnalyzers)" />
+
+  <ItemGroup Condition="$(EnableNETAnalyzers)" />
```

```suggestion
  <ItemGroup Condition="$(EnableNETAnalyzers)">
```

Extra slash is causing syntax error:

> error MSB4024: The imported project file "/datadisks/disk1/workspace/_work/1/s/artifacts/bin/redist/Release/dotnet/sdk/5.0.100-ci/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.VisualBasic.Analyzers.targets" could not be loaded. The 'Project' start tag on line 12 position 2 does not match the end tag of 'ItemGroup'. Line 28, position 5